app/src/main/java/com/android/partagix/model/InventoryViewModel.kt

```diff
@@ -119,6 +129,21 @@ class InventoryViewModel(
               newLoanList.forEach { loan -> updateLoan(loan) }
               updateInv(items.filter { it.idUser.equals(currentUser.uid) })
             }
+            val availability = mutableListOf<Boolean>()
+            items
+                .filter { it.idUser.equals(currentUser.uid) }
+                .forEach { item ->
+                  println(item)
+                  database.getItemUnavailability(item.id) { date ->
+                    availability.add(
+                        if (date.none { stripTime(it) == stripTime(Date()) }) {
+                          true
+                        } else {
+                          false
+                        })
```

Comment:
For better readability:

```suggestion
                    availability.add(date.none { stripTime(it) == stripTime(Date()) }))
```